### PR TITLE
[Etiqa.com.my] New ruleset

### PR DESCRIPTION
**Pull request by youdly**
https://github.com/EFForg/https-everywhere/pull/14066

### CODE REVIEW

File: src/chrome/content/rules/Etiqa.com.my.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<!--
+	For other Maybank coverage, see Maybank.com.xml
+
+
+	Non-functional subdomains:
+
+		- etiqa.com.my	(h)
```
Comment (J0WI) [inline src/chrome/content/rules/Etiqa.com.my.xml]: I got different content (404) on https

File: src/chrome/content/rules/Etiqa.com.my.xml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+		- www.etiqa.com.my	(t)
```
Comment (J0WI) [inline src/chrome/content/rules/Etiqa.com.my.xml]: I got different content (404) on https

File: src/chrome/content/rules/Etiqa.com.my.xml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	<target host="agentmail.etiqa.com.my" />
+	<target host="media.etiqa.com.my" />
+	<target host="t.etiqa.com.my" />
+	<target host="w.etiqa.com.my" />
+
+	<securecookie host=".+" name=".+" />
+
+	<!-- only MyAccount directory is functional -->
+	<rule from="^http://(www\.)?etiqa\.com\.my/MyAccount"
+		to="https://$1etiqa.com.my/MyAccount" />
```
Comment (J0WI) [inline src/chrome/content/rules/Etiqa.com.my.xml]: This won't work without having them in the targets.

File: src/chrome/content/rules/Etiqa.com.my.xml (hunk 1 above)
Comment (youdly) [reply src/chrome/content/rules/Etiqa.com.my.xml]: redirects to http for me including `www`

File: src/chrome/content/rules/Etiqa.com.my.xml (hunk 3 above)
Comment (youdly) [reply src/chrome/content/rules/Etiqa.com.my.xml]: I have to add `exclusion` with `(?!getonline|MyAccount)`.

File: src/chrome/content/rules/Etiqa.com.my.xml (hunk 1 above)
Comment (J0WI) [reply src/chrome/content/rules/Etiqa.com.my.xml]: See `http://etiqa.com.my/en/personal` and `https://etiqa.com.my/en/personal` (same for `www`)

File: src/chrome/content/rules/Etiqa.com.my.xml [hunk 4]
```diff
@@ -41,6 +41,25 @@
 
 	<securecookie host=".+" name=".+" />
 
+	<!-- only MyAccount & getonline directories are functional -->
+	<exclusion pattern="^http://(www\.)?etiqa\.com\.my/(?!getonline|MyAccount)" />
+
+		<test url="http://etiqa.com.my/" />
+		<test url="http://www.etiqa.com.my/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Etiqa.com.my.xml]: They are already tested (all targets are implicit test urls). Duplicated test urls should be avoided.

File: src/chrome/content/rules/Etiqa.com.my.xml (hunk 3 above)
Comment (J0WI) [reply src/chrome/content/rules/Etiqa.com.my.xml]: This is already covered by the rule below.

File: src/chrome/content/rules/Etiqa.com.my.xml [hunk 5]
```diff
@@ -41,6 +41,25 @@
...
+
+	<rule from="^http://(www\.)?etiqa\.com\.my/getonline"
+		to="https://$1etiqa.com.my/getonline" />
+
+		<test url="http://etiqa.com.my/getonline/asset/img/1471350479558.png" />
+		<test url="http://etiqa.com.my/getonline/asset/img/1471350479938.png" />
```
Comment (J0WI) [inline src/chrome/content/rules/Etiqa.com.my.xml]: Duplicated test urls should be avoided.

File: src/chrome/content/rules/Etiqa.com.my.xml [hunk 6]
```diff
@@ -41,6 +41,25 @@
...
 	<securecookie host=".+" name=".+" />
 
+	<!-- only MyAccount & getonline directories are functional -->
+	<exclusion pattern="^http://(www\.)?etiqa\.com\.my/(?!getonline|MyAccount)" />
+
+		<test url="http://etiqa.com.my/" />
+		<test url="http://www.etiqa.com.my/" />
+
+	<rule from="^http://(www\.)?etiqa\.com\.my/getonline"
+		to="https://$1etiqa.com.my/getonline" />
```
Comment (J0WI) [inline src/chrome/content/rules/Etiqa.com.my.xml]: This is already covered by the rule below.

File: src/chrome/content/rules/Etiqa.com.my.xml [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+<ruleset name="Etiqa Insurance">
+
+	<!-- only MyAccount & getonline directories are functional -->
+	<target host="etiqa.com.my" />
+	<target host="www.etiqa.com.my" />
+	<target host="academy.etiqa.com.my" />
+	<target host="agentmail.etiqa.com.my" />
+	<target host="media.etiqa.com.my" />
+	<target host="t.etiqa.com.my" />
+	<target host="w.etiqa.com.my" />
```
Comment (J0WI) [inline src/chrome/content/rules/Etiqa.com.my.xml]: Can you provide a test url for `t` and `w`? They don't send anything on  `$`.

Comment (youdly) [reply src/chrome/content/rules/Etiqa.com.my.xml]: Removed targets

**Pull request merged**